src/ui-components/Address.tsx

```diff
@@ -56,8 +58,12 @@ const Address = ({ address, className, displayInline, disableIdenticon, extensio
 	const router = useRouter();
 	const [username, setUsername] = useState('');
 	const [kiltName, setKiltName] = useState('');
+	const[manualUserName,setManualUserName] = useState('');
+	const[autoGeneratedUsername,setAutoGeneratedUsername] = useState('');
 
 	const encoded_addr = address ? getEncodedAddress(address, network) || '' : '';
+	const featureReleaseDate = dayjs('2023-06-08').toDate(); // Date from which web3 address is being used as username on web3 signup.
+	const ManualUsernames25char: string[] = ['marcosdanieldasilvasantos','masterquincyleroyharrison','sasha.fleur.polkaassembly'];
 
 	const fetchUsername = async () => {
```

Comment:
fetchUsername and fetchManualUsername, call the same apis twice in the same component

Reply:
variable name standards

Reply:
will fix